dashboard/app/models/video.rb

```diff
@@ -21,4 +21,47 @@ def self.setup
     end
     check_i18n_names
   end
+
+  def self.youtube_base_url
+    'https://www.youtube.com'
+  end
+
+  def self.youtube_url(code, args={})
```

Comment:
Good catch.  Now that it's in the model I can refactor it to not take `code` as a param.